isovar/dataframe_builder.py

```diff
@@ -55,8 +56,14 @@ def __init__(
             Dictionary mapping element_class field names to desired column names
             in the produced DataFrame.
 
+        extra_column_fns : dict
+            Dictionary mapping column names to functions which take a variant
+            and element and return a single value for each row.
```

Comment:
In every use of a DataFrameBuilder there's some `element_class` (which is an argument to the constructor) such as `AlleleRead`. Maybe I'll add that example to the comment.